pos_shift_partner_can_shop/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module displays in the pos whether the cooperator can shop or not.
+If the customer / cooperator cannot shop or is not set, a confirm popup
+is shown.
```

Comment:
Needs changing

Reply:
I think this is correct 🤔

Reply:
It's just that it's mentionning cooperators, and we can have shifts without cooperators no ?

Reply:
Ha yes, thank you 👍